src/components/admin/order/OrderToggleButton.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+import { Color } from '@resources/colors';
+import { rowFlex } from '@styles/flexStyles';
+import React from 'react';
+
+const StyledButton = styled.button`
+  ${rowFlex({ justify: 'center', align: 'center' })}
+  background: ${Color.KIO_ORANGE};
+  color: ${Color.WHITE};
+  border: none;
+  border-radius: 40px;
+  cursor: pointer;
+  padding: 5px 25px;
+  font-size: 16px;
+  font-family: 'LINE Seed Sans KR', sans-serif;
+  font-weight: 700;
+  height: 37px;
+
+  cursor: pointer;
+
+  &:hover {
+    background: #ff9d50;
+  }
+`;
+
+interface OrderToggleButtonProps {
+  toggleShowServedOrders: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
```suggestion
  toggleShowServedOrders: React.Dispatch<React.SetStateAction<boolean>>;
```

이거 어떤건지 설명해주실 수 있나요??

Reply:
<img width="529" alt="스크린샷 2025-05-14 오전 11 12 55" src="https://github.com/user-attachments/assets/cf3578e0-002b-41e4-841b-4d3ac2543df4" />

'서빙 완료 주문만 보기' 토글 버튼에서 서빙 완료된 주문만 보여줄지 말지를 결정하기 위한 상태를 변경하는 함수입니다.